dtfs-central-api/src/services/repositories/utilisation-reports-repo/utilisation-reports-repo.ts

```diff
@@ -73,8 +73,9 @@ const getUtilisationReportDetailsFilterFromOptions = (options?: GetUtilisationRe
   if (options.reportPeriod) {
     utilisationReportDetailsFilter.reportPeriod = { $eq: options.reportPeriod };
   }
-  if (options.reportStatuses) {
-    utilisationReportDetailsFilter.status = { $in: options.reportStatuses };
+  if (options.excludeNotUploaded && options.excludeNotUploaded === 'true') {
+    utilisationReportDetailsFilter.status = { $not: { $in: [ 'REPORT_NOT_RECEIVED' ] } };
+    utilisationReportDetailsFilter.azureFileInfo = { $not: { $eq: null } };
```

Comment:
`options` missing from the JSDoc in the `getOneUtilisationReportDetailsByBankId` function below.